### PR TITLE
Bumped dotliquid to support netstandard16

### PR DESCRIPTION
**Pull request by cemremengu**
### Prerequisites
- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/NancyFx/Nancy/pulls) open
- [x] I have verified that I am following the Nancy [code style guidelines](https://github.com/NancyFx/Nancy/blob/45238076ad0b7f6ecabd6bae8469e30458d02efe/CONTRIBUTING.md#style-guidelines)
- [x] I have provided test coverage for my change (where applicable)
### Description

Addresses #2598

### CODE REVIEW

**Comment by cemremengu** [comment]
Looks like the failing test is not related to dotliquid

File: test/Nancy.ViewEngines.DotLiquid.Tests.MSBuild/packages.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="DotLiquid" version="1.7.0" targetFramework="net45" />
+  <package id="DotLiquid" version="2.0.55" targetFramework="net452" />
```
Comment (thecodejunkie) [inline test/Nancy.ViewEngines.DotLiquid.Tests.MSBuild/packages.config]: We can bump this to `2.0.64` now

File: test/Nancy.ViewEngines.DotLiquid.Tests/project.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "dependencies": {
-        "DotLiquid": "1.7.0",
+        "DotLiquid": "2.0.55",
```
Comment (thecodejunkie) [inline test/Nancy.ViewEngines.DotLiquid.Tests/project.json]: We can bump this to `2.0.64` now

File: src/Nancy.ViewEngines.DotLiquid/project.json [hunk 3]
```diff
@@ -16,10 +16,15 @@
...
     "dependencies": {
         "Nancy": { "target": "project" },
-        "DotLiquid": "1.7.0"
+        "DotLiquid": "2.0.55"
     },
 
     "frameworks": {
+	    "netstandard1.6": {
+            "imports": [
+                "portable-net451+win8"
```
Comment (thecodejunkie) [inline src/Nancy.ViewEngines.DotLiquid/project.json]: Is this import really needed? It supports `netstandard1.3` in the package
image

File: src/Nancy.ViewEngines.DotLiquid.MSBuild/packages.config [hunk 4]
```diff
@@ -1,5 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-
 <packages>
-  <package id="DotLiquid" version="1.7.0" targetFramework="net45" />
-</packages>
+  <package id="DotLiquid" version="2.0.55" targetFramework="net452" />
```
Comment (thecodejunkie) [inline src/Nancy.ViewEngines.DotLiquid.MSBuild/packages.config]: We can bump this to `2.0.64` now

**Comment by thecodejunkie** [comment]
@cemremengu can you rebase this and fix the review comments and I'll pull it straight in? 😍

File: src/Nancy.ViewEngines.DotLiquid.MSBuild/Nancy.ViewEngines.DotLiquid.csproj [hunk 5]
```diff
@@ -91,9 +92,9 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="DotLiquid, Version=1.7.0.0, Culture=neutral, PublicKeyToken=82e46016ecf9f07c, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\DotLiquid.1.7.0\lib\NET40\DotLiquid.dll</HintPath>
+    <Reference Include="DotLiquid, Version=2.0.55.0, Culture=neutral, PublicKeyToken=82e46016ecf9f07c, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\DotLiquid.2.0.55\lib\net451\DotLiquid.dll</HintPath>
+      <Private>True</Private>
```
Comment (thecodejunkie) [inline src/Nancy.ViewEngines.DotLiquid.MSBuild/Nancy.ViewEngines.DotLiquid.csproj]: Does this need to be private?

File: test/Nancy.ViewEngines.DotLiquid.Tests.MSBuild/Nancy.ViewEngines.DotLiquid.Tests.csproj [hunk 6]
```diff
@@ -90,9 +91,9 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="DotLiquid, Version=1.7.0.0, Culture=neutral, PublicKeyToken=82e46016ecf9f07c, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\DotLiquid.1.7.0\lib\NET40\DotLiquid.dll</HintPath>
+    <Reference Include="DotLiquid, Version=2.0.55.0, Culture=neutral, PublicKeyToken=82e46016ecf9f07c, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\DotLiquid.2.0.55\lib\net451\DotLiquid.dll</HintPath>
+      <Private>True</Private>
```
Comment (thecodejunkie) [inline test/Nancy.ViewEngines.DotLiquid.Tests.MSBuild/Nancy.ViewEngines.DotLiquid.Tests.csproj]: Does this need to be private?

**Comment by thecodejunkie** [comment]
Thanks ❤️

**Pull request merged**